sockeye/inference.py

```diff
@@ -1005,6 +1028,9 @@ def __init__(self,
         self.interpolation_func = self._get_interpolation_func(ensemble_mode)
         self.beam_size = self.models[0].beam_size
         self.batch_size = self.models[0].batch_size
+        # skip softmax for a single model, but not for an ensemble
+        self.skip_softmax = self.models[0].skip_softmax
+        self.skip_topk = skip_topk
```

Comment:
maybe a final check_condition to cover the case of someone creating a `Translator` without `load_models`.

```
if skip_softmax:
  check_condition(len(self.models) == 0 and self.beam_size ==1, "...")
```